src/status_im2/contexts/activity_center/notification/mentions/view.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.activity-center.notification.mentions.view
+  (:require [clojure.string :as string]
+            [i18n.i18n :as i18n]
+            [quo.components.animated.pressable :as animation]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [status-im.utils.datetime :as datetime]
+            [status-im2.contexts.activity-center.notification.common.view :as common]
+            [status-im2.contexts.activity-center.notification.mentions.style :as style]
+            [utils.re-frame :as rf]))
+
+(defn message-body [message]
+  (let [parsed-text          (get-in message [:content :parsed-text])
+        parsed-text-children (:children (first parsed-text))]
+    (into [quo/text {:number-of-lines     2
+                     :style               style/tag-text
+                     :accessibility-label :activity-message-body
+                     :size                :paragraph-1}]
+          (map-indexed (fn [index {:keys [type literal]}]
+                         ^{:key index}
+                         (case type
+                           "mention" [quo/text  {:style style/mention-text
+                                                 :size  :paragraph-1}
+                                      (str "@" (rf/sub [:contacts/contact-name-by-identity literal]))]
+                           literal)) parsed-text-children))))
+
+(defn view
+  [{:keys [author chat-name chat-id message] :as notification}]
+  [animation/pressable
```

Comment:
we shouldn't use `animation/pressable` . just `touchable` from rn

Reply:
We might need a separate issue to track this because most of the notification items (Contact Request, Identification Request, Mentions, Ad-Hoc Chat and Replies) use `animation/pressable`. We need to update those as well.

Reply:
Great, didn't know that :+1:

Reply:
> We might need a separate issue to track this because most of the notification items (Contact Request, Identification Request, Mentions, Ad-Hoc Chat and Replies) use `animation/pressable`. We need to update those as well.

no need to track this, all new code should be done with quo2, we don't have pressable in quo2

Reply:
On the topic of using RN Touchable* components, the docs recommend using the new Pressable component (not to confuse with our own animation/pressable, that's another thing).

Shouldn't we also eventually migrate away from RN Touchable components? @flexsurfer

Reply:
> We might need a separate issue to track this because most of the notification items (Contact Request, Identification Request, Mentions, Ad-Hoc Chat and Replies) use `animation/pressable`. We need to update those as well.

@smohamedjavid, I'm updating them all in this PR.

Reply:
good point, we should have some guidelines from design team regarding touchables